sycl/include/sycl/ext/oneapi/matrix/matrix-unified.hpp

```diff
@@ -70,7 +69,7 @@ struct joint_matrix {
   joint_matrix() {
 #ifndef __SYCL_DEVICE_ONLY__
     throw runtime_error("joint matrix is not supported on host device.",
-                        PI_ERROR_INVALID_DEVICE);
+                        UR_RESULT_ERROR_INVALID_DEVICE);
```

Comment:
These changes should not be needed, after https://github.com/intel/llvm/pull/14485 is merged